junebug/stores.py

```diff
@@ -1,6 +1,17 @@
 from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.web import http
+
 from vumi.message import TransportUserMessage
 
+from junebug.error import JunebugError
+
+
+class MessageNotFound(JunebugError):
+    '''Raised when a message is not found.'''
+    name = 'MessageNotFound'
+    description = 'message not found'
+    code = http.NOT_FOUND
```

Comment:
+1 on 400.

Ya, wasn't too sure where to put the error. I put it in `stores` in case something else irrelevant to `channel` was using `stores` needed to raise such an error. Happy to move it into `channel` though.

Might be a sign that we need to raise an error in the stores methods (counter to what I suggested in an earlier PR review), but not too sure, still might seem weird to have some of our methods throwing errors and some returning `None`.